packages/core/android/src/main/kotlin/com/datadog/reactnative/FrameRateProvider.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import com.facebook.react.modules.core.ChoreographerCompat
+
+internal class FrameRateProvider(
+    reactFrameRateCallback: ((Double) -> Unit)?
```

Comment:
it seems that this callback value is never `null`, so we can change type to the non-nullable.